_includes/footer.html

```diff
@@ -0,0 +1,80 @@
+<footer class="footer grid-container">
```

Comment:
We may need to adjust where exactly the grid is applied in the markup to allow the background color to take effect on the full width. Right now, the footer background doesn't extend the full width on my large monitor:

![image](https://github.com/GSA-TTS/identity-dev-docs/assets/1779930/7cffac89-68e5-4ed5-aae0-093e681ae935)

Reply:
huh, I may need to requisition a larger monitor.